bigbluebutton-html5/imports/ui/components/presentation/component.jsx

```diff
@@ -154,13 +155,20 @@ class Presentation extends PureComponent {
       presentationBounds,
       numCameras,
       intl,
+      multiUser,
+      clearFakeAnnotations,
     } = this.props;
 
     const {
       numCameras: prevNumCameras,
       presentationBounds: prevPresentationBounds,
     } = prevProps;
 
+    if (!multiUser) {
```

Comment:
FYI we might want to improve this check since the cleanup was added to a component update procedure.
Preferably check whether multiUser actually transitioned states (true -> false) instead of always running this thing?

This causes problems like https://github.com/bigbluebutton/bigbluebutton/issues/15238#issuecomment-1167309254 (and potentially others since this is race-condition prone).